lib/learn_to_rank/ctr_to_judgements.rb

```diff
@@ -0,0 +1,67 @@
+module LearnToRank
+  class CtrToJudgements
```

Comment:
Is this based on the "Unbiased Learning-to-Rank from biased feedback data." mentioned in the https://github.com/tensorflow/ranking README?  If not, why not?

Reply:
No it was a hack to get scores from CTRs. The reason was for experimentation - to see how the model would react to some user-generated data (even bad data as this is).

My plan is to get this PR into a mergeable state (e.g. robust, tested, but with hacks like this). We may eventually remove this class entirely, and replace it with the usage of a click model or something else. We need to get some reliable user generated ratings we can use for training the model. But I think having something at this level of hackiness as a starting point will be useful.